test/loader.test.js

```diff
@@ -127,3 +127,38 @@ test.cb(
     });
   },
 );
+
+test.cb("should load ESM config files", t => {
```

Comment:
Can we differentiate this test based on `process.version`?

Reply:
I was thinking about it, but I don't know if Node team will backport ESM to 12 or even 10 in the future and this test begin to fail. But okay, I'll redo it with semver check of Node version. As I remember `13.3` was the first one without `--experimental` flag.

Reply:
Let's only test in on 13.3 for now. If they'll backport it we can update the test.

Reply:
Done